storybook/stories/API/cartesian/Line.stories.tsx

```diff
@@ -103,3 +173,42 @@ export const CustomizedLabel = {
     await expect(getAllByText(/Customized Label/)).toHaveLength(5);
   },
 };
+
+export const Points = {
+  render: (args: Record<string, any>) => {
+    const { points } = args;
+
+    const [surfaceWidth, surfaceHeight] = [600, 300];
+
+    return (
+      <ResponsiveContainer width="100%" height={surfaceHeight}>
+        <Surface
+          width={surfaceWidth}
+          height={surfaceHeight}
+          viewBox={{
+            x: 0,
+            y: 0,
+            width: surfaceWidth,
+            height: surfaceHeight,
+          }}
+        >
+          <Line isAnimationActive={false} points={points} />
+        </Surface>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    points: coordinateData,
+  },
+  parameters: {
+    controls: { include: ['points'] },
+    docs: {
+      description: {
+        story:
+          'You can directly set the x and y coordinates of a Line via points. This overrides dataKey and data.' +
+          'The coordinate system of the points lies in the top right of the bounding box.' +
+          'Using points, a Line can even be used within only a Surface, without a Chart.',
```

Comment:
This results in: "You can directly set the x and y coordinates of a Line via points. This overrides dataKey and data.The coordinate system of the points lies in the top right of the bounding box.Using points, a Line can even be used within only a Surface, without a Chart."

We should add trailing spaces here to alleviate this issue.

```suggestion
          'You can directly set the x and y coordinates of a Line via points. This overrides dataKey and data. ' +
          'The coordinate system of the points lies in the top right of the bounding box. ' +
          'Using points, a Line can even be used within only a Surface, without a Chart.',
```